src/components/Dissolution/AssociationDetails.vue

```diff
@@ -118,20 +118,11 @@ import { isEmpty } from 'lodash'
   }
 })
 export default class AssociationDetails extends Mixins(CommonMixin, EnumMixin, DateMixin) {
-  @Prop({ default: false })
-  readonly isSummary: boolean
-
-  @Prop({ default: 'Address' })
-  readonly addressLabel: string
-
-  @Prop({ default: 'Company' })
-  readonly entityLabel: string
-
-  @Prop({ default: false })
-  readonly showBusinessDate: boolean
-
-  @Prop({ default: true })
-  readonly showContactInfo: boolean
+  @Prop({ default: false }) readonly isSummary!: boolean
```

Comment:
The [TypeScript non-null assertion operator](https://www.logicbig.com/tutorials/misc/typescript/non-null-assertion-operator.html) is required for props (and getters and actions) so that the compiler knows this value will be provided at runtime. Without this, some code reported errors.